chromium_src/components/component_updater/component_installer.cc

```diff
@@ -1,8 +1,49 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/component_updater/component_installer.h"
+
+#include <algorithm>
 
 #include "components/crx_file/crx_verifier.h"
 #define CRX3_WITH_PUBLISHER_PROOF CRX3
+#define Register Register_ChromiumImpl
 #include "../../../../components/component_updater/component_installer.cc"
+#undef Register
 #undef CRX3_WITH_PUBLISHER_PROOF
+
+namespace component_updater {
+
+void ComponentInstaller::Register(ComponentUpdateService* cus,
+                                  base::OnceClosure callback) {
+  static std::vector<std::string> blacklisted_components({
+      "bklopemakmnopmghhmccadeonafabnal",  // Legacy TLS Deprecation Config
+      "cmahhnpholdijhjokonmfdjbfmklppij",  // Federated Learning of Cohorts
+      "eeigpngbgcognadeebkilcpcaedhellh",  // Autofill States Data
+      "gcmjkmgdlgnkkcocmoeiminaijmmjnii",  // Subresource Filter Rules
+      "ggkkehgbnfjpeggfpleeakpidbkibbmn",  // Crowd Deny
```

Comment:
I think you can remove this to fix: https://github.com/brave/brave-browser/issues/10280

Reply:
Thanks @jumde. I'm afraid I don't understand this suggestion. The issue you linked to is about automatic updates failing for Crowd Deny. The line you refer to prevents Crowd Deny from being installed. This is in the spirit of this PR, because it follows the rule "remove all components that appear with `0.0.0.0`, of which Crowd Deny is one. If I remove the line, then Crowd Deny will still appear, and fail to auto-update until we merge #7167.

Can you explain a bit further? I'm of course happy to follow any instructions; If you tell me to remove the line, I will of course. But I'd love to understand the reason. Thanks!

Reply:
@mherrmann - I think you can keep the `crowd deny` extension in the list but you should remove `Resolves brave/brave-browser#10280` in the PR description.

Reply:
I've removed the `Resolves brave/brave-browser#10280` in the PR description.